.env

```diff
@@ -1,2 +1 @@
-
-password=pk.e1f8f92e635d436ee387f6d6afb1d612
\ No newline at end of file
+REACT_APP_LOCATION_KEY=pk.e1f8f92e635d436ee387f6d6afb1d612
```

Comment:
We will need to add this file to `.gitignore` since it is still showing as committed 👍